documentation/src/main/asciidoc/topics/proc_using_jgroups_default.adoc

```diff
@@ -0,0 +1,51 @@
+[id='jgroups_getting_started-{context}']
+= Getting Started with Default Stacks
+{brandname} uses JGroups protocol stacks so nodes can send each other messages on dedicated cluster channels.
+
+{brandname} provides preconfigured JGroups stacks for `UDP` and `TCP` protocols.
+You can use these default stacks as a starting point for building custom cluster transport configuration that is optimized for your network requirements.
+
+.Procedure
+
+. Locate the default JGroups stacks in the `default-configs` directory inside the `infinispan-core-{ispn_version}.jar` file.
```

Comment:
not sure why step 1 (locate) & 2 (view xml) are required. Reading this it seems mandatory to open the XML file. 
IMO, You can just tell where the XML is.